test/e2e/testing_parameters.go

```diff
@@ -0,0 +1,19 @@
+package e2e
+
+import "flag"
+
+var (
+	isProductBuild bool
+)
+
+func init() {
+	flag.BoolVar(&isProductBuild, "product", false, "Using RHSSO or Keycloak")
+}
+
+func currentProfile() string {
+	if isProductBuild {
+		return "RHSSO"
```

Comment:
I will change this to a variable once https://github.com/keycloak/keycloak-operator/pull/165 is merged. Importing common package at the moment is causing issues with parameter parsing I mentioned you on the call.